sktime/classification/deep_learning/cnn.py

```diff
@@ -105,6 +109,8 @@ def __init__(
         optimizer=None,
         filter_sizes=None,
         padding="auto",
+        compile_kwargs: KerasCompileKwargs = KerasCompileKwargs(),
```

Comment:
avoid mutable defaults - this is a general python best practice. Instead default `None` and replace internally.